source/includes/steps-create-index.yaml

```diff
@@ -27,18 +27,55 @@ content: |
 
```

Comment:
I don't know how to comment on the rest of this file? But Step 3 is missing a period between the two sentences.

Reply:
Fixed!